Makefile

```diff
@@ -27,6 +27,8 @@ check-dictionaries:
 	done
 	@if command -v pytest > /dev/null; then \
 		pytest codespell_lib/tests/test_dictionary.py; \
+	else \
+		echo "pytest command not present, dictionary tests being skipped"; \
```

Comment:
I dunno if this should exit or just warn? I guess technically it's "some dictionary tests" being skipped, as trim and sort are being run.

Reply:
Might be worth adding `1>&2` on the end to get it echoing to STDERR instead of STDOUT?

Reply:
Yeah I could do, although the other errors in the Makefile are going to STDOUT currently. Personally I'd be tempted to drop the @'s so it generates more output, but I guess that might be too noisy for some people's preference. It certainly would have made finding this issue easier.

Reply:
I would probably make it exit with an informative error so that people are pushed toward installing the necessary dependencies

Reply:
Done.